docker/build_image.sh

```diff
@@ -39,6 +40,7 @@ do
           echo "-n, --nightly specify to build with TorchServe nightly"
           echo "-s, --source specify to build with TorchServe from source"
           echo "-r, --remote specify to use local TorchServe"
+          echo "-a, --arch to arm64"
```

Comment:
This should be generic .. Ex: --arch="linux/aarch64" or if someone wants to build for a new platform they can use --arch="new platform"

Reply:
also please check this. May be it can be built in a single command like here https://github.com/pytorch/serve/pull/3162/files